File: CPP/Arreglos/Arreglos-2.cpp
#include <iostream>
#include <stdio.h>
using namespace std;
int main() {
 int catNumeros = 0;
 int numeros[catNumeros];
 cout<<"Ingrese tamaño del arreglo: ";
 cin>>catNumeros;
 for (int i = 0; i < catNumeros; i++){
   //numeros[i] = i + 1;
   cout<<"Ingrese el dato: "<<(i + 1)<<endl;
   cin>>numeros[i];
 }
  // mostrar contenido del arreglo
cout<<"Los numeros ingresados son: \n";

 for (int i = 0; i < catNumeros; i++){
   cout<<numeros[i]<<"\n";
 }
 }

File: Python/Arreglos/Arreglos-2.py
cantNotasString = input("Ingrese el tamano del arreglo: ")

cantNotas = int(cantNotasString)
notas = []
for i in range(0, cantNotas):
    notas.append(float(input("Ingrese el numero " + str(i + 1) + ": ")))
